### ISSUE TITLE
How to set different lr_config for some of the parameters?

### ISSUE THREAD

**Issue by thss15fyt** [open-mmlab/mmcv#2297]
The `lr_config` indicates the training scheduler of how the learning rate varies throughout training. If I want to set different `lr_config` for some parameters of the whole model (e.g. fixed lr without decaying), what should I do? Thanks for your support.

**Comment by HAOCHENYE** [comment]
Hi ! We can config optimizer like:

```python
embed_multi = dict(lr_mult=1.0, decay_mult=0.0)
norm_multi = dict(lr_mult=1.0, decay_mult=0.0)
custom_keys = {
    'norm': norm_multi,
    'absolute_pos_embed': embed_multi,
    'relative_position_bias_table': embed_multi,
    'query_embed': embed_multi
}

optim_wrapper = dict(
    optimizer=dict(lr=6e-5, weight_decay=0.01),
    paramwise_cfg=dict(custom_keys=custom_keys, norm_decay_mult=0.0))
```

BTW, recently we update our training architecture, and have released [`MMEngine`](https://github.com/open-mmlab/mmengine), we have more ['detailed docs'](https://mmengine.readthedocs.io/zh_CN/latest/tutorials/optim_wrapper.html#id7) for how to config the optimizer. We highly recommended you to experience `MMEngine`, it is more friendly, more efficient, more powerful!
